src/codegen/templates/library_interpreter_templates/signal_event_function_call.py.mako

```diff
@@ -0,0 +1,42 @@
+<%page args="function"/>\
+<%
+    from codegen.utilities.function_helpers import get_arguments_type
+    from codegen.utilities.interpreter_helpers import generate_interpreter_function_call_args, get_callback_param_data_types
+    from codegen.utilities.text_wrappers import wrap
+%>
+<% 
+callback_func_param = ""
+%>\
+%for parameter in function.base_parameters:
+    %if parameter.parameter_name == "callback_function":
+        ${parameter.type} = ctypes.CFUNCTYPE(
+            ${', '.join(get_callback_param_data_types(parameter.callback_params)) | wrap(24, 24)})        
```

Comment:
The numbers you pass to the wrap() function should match the indentation level that you want. For example, this line is indented by 16 spaces, so use wrap(16, 16) to indent the first line by 16 spaces and the following lines by 16 spaces. Omit the second number to wrap all of the lines the same.

(The first line's indentation is really done by the mako template, but wrap needs to know how much you indented by so that it can wrap the first line after the correct number of characters.)

This affects every wrap() call in this file.

Reply:
They are still incorrect.

Reply:
Updated to wrap(12)